devel/aspcud/Portfile

```diff
@@ -1,40 +1,40 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem                1.0
-PortGroup           cmake 1.0
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
 PortGroup           boost 1.0
 
-name                aspcud
-version             1.9.1
-revision            3
+github.setup        potassco aspcud 1.9.5 v
 categories          devel math
 platforms           darwin
 maintainers         nomaintainer
-license             GPL-3
+license             MIT
 description         CUDF package dependency solver
 long_description \
     Aspcud is a solver for package dependencies. A package universe and a request \
     to install, remove, or upgrade packages have to be encoded in the CUDF format. \
     Such a CUDF document can then be passed to aspcud along with an optimization \
     criteria to obtain a solution to the given package problem.
 
-homepage            http://potassco.sourceforge.net
-master_sites        sourceforge:project/potassco/aspcud/${version}/
+homepage            https://potassco.org/aspcud/
+github.tarball_from archive
+worksrcdir          ${name}-${version}
 
-distname            ${name}-${version}-source
+checksums           rmd160  5989503afe812ba48af6f15e2308b0f0c78d7283 \
+                    sha256  9cd3a9490d377163d87b16fa1a10cc7254bc2dbb9f60e846961ac8233f3835cf \
+                    size    70073941
 
-checksums           rmd160  435c80ef6879b2e049caf0e1c4d5bf2d8165b4f2 \
-                    sha256  e0e917a9a6c5ff080a411ff25d1174e0d4118bb6759c3fe976e2e3cca15e5827
-
-boost.version       1.71
+boost.version       1.76
 boost.depends_type  build
 
-depends_build-append \
-                    port:re2c
-depends_lib         port:clasp \
-                    port:gringo
+compiler.cxx_standard   2014
 
-cmake.out_of_source yes
+depends_build-append port:re2c
+depends_lib          port:clingo
```

Comment:
this line and the one above do not seem to conform to the modeline (i.e., multiples of four spaces indentation)